src/app/CommandHandler.h

```diff
@@ -249,6 +249,7 @@ class CommandHandler
     void Close();
 
     CHIP_ERROR ProcessCommandDataIB(CommandDataIB::Parser & aCommandElement);
+    CHIP_ERROR ProcessGroupCommandDataIB(CommandDataIB::Parser & aCommandElement);
```

Comment:
Please document that `ProcessCommandDataIB` is only called when we are a unicast invoke (and hence expects endpoint ids in its command paths), while `ProcessGroupCommandDataIB` is called for a group invoke and gets the group from the message metadata.

Reply:
done.